### PR TITLE
KK-299 | Add special mobile display of event occurrence list

### PR DESCRIPTION
**Pull request by badrange**

### CODE REVIEW

File: src/domain/event/eventOccurrence.module.scss [hunk 1]
```diff
@@ -8,16 +8,46 @@
...
+    grid-column: 1 / span 3;
+  }
+  td.occurrenceDate {
+    grid-row: 2;
+    grid-column: 1 / span 2;
+  }
+  td.occurrenceTime {
+    grid-row: 2;
+    grid-column: 2 / span 2;
+  }
```
Comment (jpsalo) [inline src/domain/event/eventOccurrence.module.scss]: Date and time starts to overlap at around 400px width. Is it something we need to consider/support?
localhost_3000_fi_event_RXZlbnROb2RlOjE=

Comment (badrange) [reply src/domain/event/eventOccurrence.module.scss]: Do any device exist that have a 400px screen width?

Comment (badrange) [reply src/domain/event/eventOccurrence.module.scss]: Let me see if we can find a better solution

Comment (badrange) [reply src/domain/event/eventOccurrence.module.scss]: Does it work better now?

**Pull request merged**